src/js/protocols/devices.js

```
@@ -72,11 +73,12 @@ export const usbDevices = {
 };
 
 export const vendorIdNames = {
-    1027: "FTDI",
-    1155: "STM Electronics",
-    4292: "Silicon Labs",
+    0x0403: "FTDI",
```

Biome (1.9.4)

[error] Hexadecimal number literal is not allowed here.

Safe fix: Replace 0x0403 with 1027


(lint/complexity/useSimpleNumberKeys)